RecastDemo/Source/Sample.cpp

```diff
@@ -42,7 +42,10 @@ Sample::Sample() :
 	m_crowd(0),
 	m_navMeshDrawFlags(DU_DRAWNAVMESH_OFFMESHCONS|DU_DRAWNAVMESH_CLOSEDLIST),
 	m_tool(0),
-	m_ctx(0)
+	m_ctx(0),
+	m_filterLowHangingObstacles(true),
```

Comment:
Nitpick: these should go between `m_navMeshDrawFlags` and `m_tool` to match member ordering in the class.